modules/viewing-the-log-analytics-to-detect-threats.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * integration/integrating-with-microsoft-sentinel-notifier.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-the-log-analytics-to-detect-threats_{context}"]
+= Viewing the log analytics to detect threats
+
+By creating a Microsoft sentinel integration, you can view the log analytics to detect threats.
+
+.Prerequisites
+
+* You have created a data collection rule, log analytics workspace, service principal and sentinel on Microsoft Azure.
+* You have configured the notifier supports secret or client certificate authentication.
+* You have created a schema.
```

Comment:
```suggestion
* You have created a log analytics schema with two fields following format. We recommend separate log analytics tables for Audit Logs and Alerts, both data sources use the same schema.

   * `TimeGenerated` - Datetime
   * `msg` - Dynamic

. Example JSON: 
* ```
"TimeGenerated": "2024-09-03T10:56:58.5010069Z",
"msg": {
     "id": "uuid string"
}
```

Reply:
@SimonBaeumer I think we do this step by uploading the schema to Microsoft Sentinel which is in the Azure portal. If so, we do not need to document this step in our docs. We could just a cross-ref to Azure docs.
Is there a cross-ref link that you could help me with?

Reply:
It is true that the user needs to upload the schema in Azure, but we define it. Without us documenting the schema the user does not know what to upload to Azure.